### PR TITLE
Add pytest fixture support

### PR DESCRIPTION
**Pull request by belgeu**
Currently test has no setup/teardown workflow.

Add pytest's builtin fixture support in order to provide ability to
execute [setup] and [teardown] steps before and after each test case run.

### CODE REVIEW

File: python/webdriver_test.py [hunk 1]
```diff
@@ -27,20 +27,25 @@
...
 from selenium import webdriver
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support.ui import WebDriverWait
 from selenium.webdriver.support import expected_conditions as EC
 
-def test_01():
-    b = webdriver.Firefox()
+@pytest.fixture
+def setup(request):
+    request.b = webdriver.Firefox()
```
Comment (extsoft) [inline python/webdriver_test.py]: Don't add driver to requests.  Use 
```
driver = webdriver.Firefox()
.....
return driver
```

File: python/webdriver_test.py [hunk 2]
```diff
@@ -27,20 +27,25 @@
...
+import pytest
 from selenium import webdriver
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support.ui import WebDriverWait
 from selenium.webdriver.support import expected_conditions as EC
 
-def test_01():
-    b = webdriver.Firefox()
+@pytest.fixture
+def setup(request):
```
Comment (extsoft) [inline python/webdriver_test.py]: Pls rename fixture (`browser` etc.)

File: python/webdriver_test.py (hunk 1 above)
Comment (extsoft) [inline python/webdriver_test.py]: Don't add driver to requests.  Use 
```
driver = webdriver.Firefox()
.....
return driver
```

**Review by extsoft — CHANGES_REQUESTED** [review]

File: python/webdriver_test.py (hunk 2 above)
Comment (belgeu) [reply python/webdriver_test.py]: Changed fixture name to 'browser'.

File: python/webdriver_test.py (hunk 1 above)
Comment (belgeu) [reply python/webdriver_test.py]: Modified fixture to return webdriver object separately from request.

File: python/webdriver_test.py [hunk 3]
```diff
@@ -27,20 +27,25 @@
...
+@pytest.fixture
+def browser(request):
+    driver = webdriver.Firefox()
+    def teardown():
+        driver.quit()
+    request.addfinalizer(teardown)
+    return driver
+
+def test_01(browser):
+    b = browser
```
Comment (extsoft) [inline python/webdriver_test.py]: Why do you create this `b`?

**Review by extsoft — CHANGES_REQUESTED** [review]

File: python/webdriver_test.py (hunk 3 above)
Comment (belgeu) [reply python/webdriver_test.py]: The idea is to keep code a bit shorter, but i see your point. Switched webdriver naming to "browser", it certainly improves code readability. Thanks.

**Review by extsoft — APPROVED** [review]

**Pull request merged**